Tests/PerformanceTests.swift

```diff
@@ -0,0 +1,124 @@
+// Created by Cal Stephens on 2/14/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Foundation
+import XCTest
+
+@testable import Lottie
+
+// MARK: - PerformanceTests
+
+final class PerformanceTests: XCTestCase {
+
+  // MARK: Internal
+
+  func testAnimationViewSetup_simpleAnimation() {
+    // Compare the performance of displaying this simple animation in the two animation engines
+    let ratio = compareEngineSetupPerformance(for: simpleAnimation, iterations: 2000)
+
+    // This is basically a snapshot test for the performance of the Core Animation engine
+    // compared to the Main Thread engine. Currently, the Core Animation engine is
+    // about 1.6x slower than the Main Thread engine in this example.
+    XCTAssertEqual(ratio, 1.6, accuracy: 0.2)
```

Comment:
bumping this to give more leeway in CI

```suggestion
    XCTAssertEqual(ratio, 1.6, accuracy: 0.3)
```